datashape/util/tests/test_testing.py

```diff
@@ -190,3 +190,33 @@ def test_nested():
         )
     assert "'int32' != 'float32'" in str(e.value)
     assert "_.measure['a'].measure['b'].name" in str(e.value)
+
+
+def test_function():
+    base = dshape('(int32, int64) -> int64')
+    assert_dshape_equal(base, base)
+    cases = (
```

Comment:
I didn't want to re-test the base case case over an over. I can split it out into two functions